@here/harp-mapview/lib/PickHandler.ts

```diff
@@ -82,6 +82,11 @@ export interface PickResult {
      */
     dataSourceName: string | undefined;
 
+    /**
+     * Data source order, useful for sorting a collection of picking results.
```

Comment:
Please add a comment that objects added via the `mapAnchor` API on `MapView` will have an undefined value here, because there is no such order.

Reply:
But is it expected to have such objects treated as belonging to the base layer during picking? Or they should be returned first instead, like text labels?

Reply:
I'm not so sure. 

You make the assumption anyway now that they belong to a datasource with order 0, so it doesn't hurt to add it, but yes, we can think about this.

Reply:
I agree, added a comment. Will update it if we decide to sort map anchor objects first.